test/integration/008_schema_tests_test/models/schema.yml

```diff
@@ -29,7 +29,7 @@ table_summary:
             - { field: favorite_color, values: ['blue', 'green'] }
 
         relationships:
-            - { from: favorite_color, to: table_copy, field: favorite_color }
+            - { from: favorite_color, to: ref('table_copy'), field: favorite_color }
```

Comment:
ahh i understand now. is this a breaking change in how schema tests are defined?